osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/alarm/ClearAlarmRegisterCommandExecutor.java

```diff
@@ -68,11 +70,11 @@ public AccessResultCode execute(final DlmsConnectionManager conn, final DlmsDevi
 
         final SetParameter setParameter = this.getSetParameter();
 
-        conn.getDlmsMessageListener().setDescription("ClearAlarmRegister, with alarm code = " + ALARM_CODE
-                + "and set attribute: "
-                + JdlmsObjectToStringUtil.describeAttributes(new AttributeAddress(CLASS_ID, OBIS_CODE, ATTRIBUTE_ID)));
+        conn.getDlmsMessageListener().setDescription(
+                "ClearAlarmRegister, with alarm code = " + ALARM_CODE + "and set attribute: " + JdlmsObjectToStringUtil
```

Comment:
Replace string concatenation with String.format (there is a standard refactor action for this in IntelliJ)